tools/integration_tests/readonly/create_and_delete_object_test.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2023 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Provides integration tests for file operations with --o=ro flag set.
+package readonly_test
+
+import (
+	"io/fs"
+	"os"
+	"path"
+	"testing"
+
+	"github.com/googlecloudplatform/gcsfuse/tools/integration_tests/setup"
+)
+
+func createFile(filePath string, t *testing.T) {
+	file, err := os.OpenFile(filePath, os.O_CREATE, setup.FilePermission_0600)
```

Comment:
What happens if file already exist and trying to open it to append some data?

Shouldn't we add os.O_APPEND flag?

Reply:
Shouldn't we do that in write tests?

Reply:
I am about to adding more tests for read and write also